schemas/triniti/notify/mixin/notification/1-0-0.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:triniti:notify:mixin:notification:1-0-0" mixin="true" extends="gdbots:ncr:mixin:node:v1">
+    <fields>
+      <field name="_id" type="identifier" required="true" overridable="true">
+        <php-options>
+          <imports>use Triniti\Schemas\Notify\NotificationId;</imports>
+          <class-name>NotificationId::class</class-name>
+          <default>function() { return NotificationId::generate(); }</default>
+        </php-options>
+        <js-options>
+          <imports>import NotificationId from '@triniti/schemas/triniti/notify/NotificationId';</imports>
+          <class-proto>NotificationId</class-proto>
+          <default>() => NotificationId.generate()</default>
+        </js-options>
+      </field>
+      <field name="app_ref" type="identifier">
+        <description>
+          A reference to the app to send the notification.
+        </description>
+        <php-options>
+          <imports>use Gdbots\Schemas\Ncr\NodeRef;</imports>
+          <class-name>NodeRef::class</class-name>
+        </php-options>
+        <js-options>
+          <imports>import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';</imports>
+          <class-proto>NodeRef</class-proto>
+        </js-options>
+      </field>
+      <field name="customize_alert_text" type="text" max="5000">
+        <description>
+          A customize alert text of the notification. It should typically
+          not have HTML.
+        </description>
+      </field>
+      <field name="message_type" type="string-enum">
+        <default>unknown</default>
+        <enum id="triniti:notify:message-type"/>
+      </field>
+      <field name="target_ref" type="identifier">
+        <description>
+          A reference to the target message for the notification.
+        </description>
+        <php-options>
+          <imports>use Gdbots\Schemas\Ncr\NodeRef;</imports>
+          <class-name>NodeRef::class</class-name>
+        </php-options>
+        <js-options>
+          <imports>import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';</imports>
+          <class-proto>NodeRef</class-proto>
+        </js-options>
+      </field>
+      <field name="notification_status" type="string-enum">
```

Comment:
@weiwang314 should be send_status and also need a sent_at date-time, similar to published_at on publishable